README.md

```diff
@@ -70,9 +83,11 @@ fragment FullType on __Type {
 You can now pipe that file to the `query` command to send it.
```

Comment:
Unclear what file I'm piping to the `query` command here now--why was the example removed? At the least we should say something like "If you have a query in the file `query.gql`, you can pipe that file ..."